charts/primary-site/values.yaml

```diff
@@ -1,3 +1,8 @@
 globals:
   site_token:
```

Comment:
siteToken

Reply:
I don't think this should be set here - the site token will be defined in a secret, not a yaml file

Reply:
That is not correct - the siteToken is specified during the initial install. See our installation guide. It is part of the helm variables.